packages/consumption/test/modules/requests/OutgoingRequestsController.test.ts

```diff
@@ -224,6 +230,56 @@ describe("OutgoingRequestsController", function () {
                 message: "You cannot share a Request with yourself."
             });
         });
+
+        test("returns a validation result that contains an error for requests that would lead to the creation of more than one RelationshipAttribute with the same key", async function () {
+            const validationResult = await When.iCallCanCreateForAnOutgoingRequest({
+                content: {
+                    items: [
+                        CreateAttributeRequestItem.from({
+                            mustBeAccepted: true,
+                            attribute: RelationshipAttribute.from({
+                                "@type": "RelationshipAttribute",
+                                owner: "did:e:a-domain:dids:anidentity",
+                                key: "uniqueKey",
```

Comment:
Would `"duplicateKey"` be a better naming? Same in other places.

Reply:
I think `"uniqueKey"` in connection with the name `error.consumption.requests.violatedKeyUniquenessOfRelationshipAttributes` of the error code is better. 😄 But I can also change it if it's important to you. 🎁

Reply:
Na, it's not too important to me. :)